co-log-benchmark/Main.hs

```diff
@@ -106,3 +109,13 @@ timeProcess n = do
       runProcess_ cfg
   t' <- getCurrentTime
   pure $ t' `diffUTCTime` t
+
+-- |  Function that takes list of pairs - benchmark name
+-- and result and generates markdown table.
```

Comment:
It would be much better if `main` uses this `genTable` function to not just print the results of current benchmarks to the terminal, but also collect all results and only then output them as markdown table. So later everyone can just run the benchmarks and copy-paste the table to README.